src/Backups/BackupImpl.cpp

```diff
@@ -81,7 +81,9 @@ BackupImpl::BackupImpl(
     const std::optional<BackupInfo> & base_backup_info_,
     std::shared_ptr<IBackupReader> reader_,
     const ContextPtr & context_)
-    : backup_name_for_logging(backup_name_for_logging_)
+    : context(context_)
+    , has_throttler(static_cast<bool>(context->getBackupsThrottler()))
```

Comment:
It doesn't feel like it should be here. Can we move `has_throttler` to `BackupWriter*` classes?

Reply:
Ok, done.